backend/core/src/application-flagged-sets/application-flagged-sets.service.ts

```diff
@@ -33,10 +33,7 @@ export class ApplicationFlaggedSetsService {
   async listPaginated(queryParams: PaginatedApplicationFlaggedSetQueryParams) {
     const results = await paginate<ApplicationFlaggedSet>(
       this.afsRepository,
-      {
-        limit: queryParams.limit > 0 ? queryParams.limit : Number.MAX_SAFE_INTEGER,
-        page: queryParams.page > 0 ? queryParams.page : 1,
-      },
```

Comment:
IIUC we're relying on the default value if these params aren't present? But we're now allowing the user to request page -1 (if it's in the URL for example) by not doing validation here as well?

Reply:
The pagination library checks for negative integers here:
https://github.com/nestjsx/nestjs-typeorm-paginate/blob/238190de1d7f77af0604d6741a9b4d38cbcb665e/src/paginate.ts#L128-L129, so we don't need to do it.

Context:
Andy added this (and others like it) when we set the PaginationQueryParams defaults to `-1`, to mean "get everything".  That meant we had to convert `-1` into something the pagination library could understand here.

In https://github.com/bloom-housing/bloom/pull/1578/files#diff-06004b3008e3dd6149854cbaebc150f047c6e8bdbff2dfa0818e2a73bea323d3 I set the default back to `10` to match upstream, which means we don't need to do this conversion here anymore, so I reverted this to match upstream as well.

If we wanted we could update the pagination validators to check that the provided value is a positive integer, but that should be a different PR that we can merge upstream.